File: museum-backend/api/models/Event.js
/**
 * Event.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

    name : {
      type : 'string',
      minLength : 3,
      maxLength : 64
    },
    dateStart : {
      type : 'ref',
      columnType : 'date'
    },
    timeStart : {
      type : 'ref',
      columnType : 'time'
    },
    endDate : {
      type : 'ref',
      columnType : 'date'
    },
    endTime : {
      type : 'ref',
      columnType : 'time'
    },
    description : {
      type : 'ref',
      columnType: 'text',
      maxLength : 280
    },
    museum_event_FK : {
      model : 'museum'
    }

  },

};

File: museum-app/app/src/main/java/com/example/museum_app/model/Event.kt
package com.example.museum_app.model

class Event(
    var id : Int,
    var name: String,
    var dateStart:String,
    var timeStart:String,
    var endDate : String,
    var endTime : String,
    var description : String
) {

    var image : String

    init {
        image = "http://192.168.0.8:1337/pictureEvent?id=$id"
    }

}